services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -1794,6 +1793,28 @@ public Map<String, String> getFutureVersionsForMultiColos(String clusterName, St
     return result;
   }
 
+  @Override
+  public Map<String, String> getBackupVersionsForMultiColos(String clusterName, String storeName) {
+    Map<String, ControllerClient> controllerClients = getVeniceHelixAdmin().getControllerClientMap(clusterName);
+    Map<String, String> result = new HashMap<>();
+    for (Map.Entry<String, ControllerClient> entry: controllerClients.entrySet()) {
+      String region = entry.getKey();
+      ControllerClient controllerClient = entry.getValue();
+      MultiStoreStatusResponse response = controllerClient.getBackupVersions(clusterName, storeName);
```

Comment:
Why is it multi store response?

Reply:
Its a multi-store `status` response. I repurposed it because there is no `StoreStatus` response.